### PR TITLE
trt-1530: specify deployment id for unique log names

### PR DESCRIPTION
**Pull request by neisw**

### CODE REVIEW

**Comment by neisw** [comment]
/payload-job periodic-ci-openshift-release-master-ci-4.16-upgrade-from-stable-4.15-e2e-gcp-ovn-rt-upgrade

**Comment by neisw** [comment]
/payload-job periodic-ci-openshift-release-master-ci-4.16-upgrade-from-stable-4.15-e2e-gcp-ovn-rt-upgrade

**Comment by neisw** [comment]
/payload-job periodic-ci-openshift-release-master-ci-4.16-upgrade-from-stable-4.15-e2e-gcp-ovn-rt-upgrade

**Comment by neisw** [comment]
/payload-job periodic-ci-openshift-release-master-ci-4.16-upgrade-from-stable-4.15-e2e-gcp-ovn-rt-upgrade

**Comment by neisw** [comment]
https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/logs/openshift-origin-28613-ci-4.16-upgrade-from-stable-4.15-e2e-gcp-ovn-rt-upgrade/1761762293827244032/artifacts/e2e-gcp-ovn-rt-upgrade/openshift-e2e-test/artifacts/junit/e2e-monitor-tests__20240225-165813.xml

`Initializing output file to /var/log/persistent-logs/disruption-pod-to-service-network-5eed6720-8f79-496c-878f-d0b1ebd666ee.jsonl`

**Comment by neisw** [comment]
/payload-job periodic-ci-openshift-release-master-ci-4.16-upgrade-from-stable-4.15-e2e-gcp-ovn-rt-upgrade

File: pkg/cmd/openshift-tests/disruption/poll-service/poll_service_options.go [hunk 1]
```diff
@@ -32,14 +32,15 @@ type PollServiceOptions struct {
...
 
 	startingContent, err := os.ReadFile(o.OutputFile)
 	if err != nil && !os.IsNotExist(err) {
 		return err
 	}
 	if len(startingContent) > 0 {
-		//print starting content to the log so that we can simply scrape the log to find all entries at the end
-		o.OriginalOutFile.Write(startingContent)
+		// print starting content to the log so that we can simply scrape the log to find all entries at the end
+		o.OriginalOutFile.Write([]byte(fmt.Sprintf("Found existing content: %s", string(startingContent))))
```
Comment (deads2k) [inline pkg/cmd/openshift-tests/disruption/poll-service/poll_service_options.go:43]:
```suggestion
		o.OriginalOutFile.Write([]byte(fmt.Sprintf("Found existing content:\n %s", string(startingContent))))
```

**Comment by neisw** [comment]
/payload-job periodic-ci-openshift-release-master-ci-4.16-upgrade-from-stable-4.15-e2e-gcp-ovn-rt-upgrade

**Pull request merged**